src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -669,3 +670,14 @@ export const saveScraperTarget = () => async (
     console.error()
   }
 }
+
+interface SetLocationOnDismiss {
+  type: 'SET_LOCATION_ON_DISMISS'
+  payload: {locationOnDismiss: string}
+}
+export const setLocationOnDismiss = (
+  locationOnDismiss: string
+): SetLocationOnDismiss => ({
+  type: 'SET_LOCATION_ON_DISMISS',
+  payload: {locationOnDismiss},
+})
```

Comment:
Although we prefer a different pattern for the action types, we cannot use two different patterns in the same file. Refactoring the whole file just to add one action was too much to do here.

The preferred pattern can be applied in a future refactor for the whole file.